export function handleFetchError(err: any) {
    console.log(err);
}
